src/commands/commandRegistry.ts

```diff
@@ -31,21 +33,25 @@ export class CommandRegistry {
       this._commands.set(phrase, existingCommand as ICommand);
     } else {
       // create new instance of the command and map it to the phrase
-      this._commands.set(phrase, new (command as any)());
-      command.onload?.();
+      const instance = new (command as any)();
+      this._commands.set(phrase, instance);
+      instance.onLoad?.();
```

Comment:
similar to the issue below in `execute` it wasn't maintaining the correct `this` context the way it was called before.